website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -480,3 +480,19 @@ up all residual volumes that are not designated by the user to remain after
 termination. If you need to preserve those source volumes, you can overwrite the
 termination setting by specifying `delete_on_termination=false` in the
 `launch_block_device_mappings` block for the device.
+
+## Windows 2016 Sysprep Commands
+
+For Windows 2016 Images it is necessary to run Sysprep commands which can be easily added
```

Comment:
According to #6297, This isn't necessary for all Windows 2016 images, just for AWS Windows Server 2016 AMIs that come with EC2Launch pre-installed. Mind changing this to be more specific?

Reply:
I just did.